### ISSUE TITLE
Why not F.normalize() when training?

### ISSUE THREAD

**Issue by Jiahaohong** [yoonkicho/PPLR#16]
I see the code doesn't F.normalize the features after gap or rap when training, but normalizes them in function extract_all_features. So what's the difference in tranning and extract_all_features? Why not F.normalize features when training?
